src/sentry/audit_log/register.py

```diff
@@ -0,0 +1,626 @@
+            message += ": {}".format(", ".join(filter_name))
+        return message
+    return f"enabled project filter {filter_name}"
+
+
+def render_project_disable(audit_log_entry: AuditLogEntry):
+    # Most logs will just be name of the filter, but legacy browser changes can be bool, str or sets
+    filter_name = audit_log_entry.data["state"]
+    if filter_name in ("0", "1") or isinstance(filter_name, set) or isinstance(filter_name, bool):
+        message = "disabled project filter legacy-browsers"
+        if isinstance(filter_name, set):
+            message += ": {}".format(", ".join(filter_name))
+        return message
+    return f"disabled project filter {filter_name}"
+
+
+def render_sso_edit(audit_log_entry: AuditLogEntry):
+    settings = ", ".join(f"{k} {v}" for k, v in audit_log_entry.data.items())
+    return "edited sso settings: " + settings
+
+
+def render_servicehook_add(audit_log_entry: AuditLogEntry):
+    full_url = audit_log_entry.data.get("url")
+    return f'added a service hook for "{truncatechars(full_url, 64)}"'
+
+
+def render_servicehook_edit(audit_log_entry: AuditLogEntry):
+    full_url = audit_log_entry.data.get("url")
+    return f'edited the service hook for "{truncatechars(full_url, 64)}"'
+
+
+def render_servicehook_remove(audit_log_entry: AuditLogEntry):
+    full_url = audit_log_entry.data.get("url")
+    return f'removed the service hook for "{truncatechars(full_url, 64)}"'
+
+
+def render_integration_upgrade(audit_log_entry: AuditLogEntry):
+    if audit_log_entry.data.get("provider"):
+        return "upgraded {name} for the {provider} integration".format(**audit_log_entry.data)
+    return "updated an integration"
+
+
+def render_integration_add(audit_log_entry: AuditLogEntry):
+    if audit_log_entry.data.get("provider"):
+        return "installed {name} for the {provider} integration".format(**audit_log_entry.data)
+    return "enabled integration {integration} for project {project}".format(**audit_log_entry.data)
+
+
+def render_integration_edit(audit_log_entry: AuditLogEntry):
+    if audit_log_entry.data.get("provider"):
+        return "edited the {name} for the {provider} integration".format(**audit_log_entry.data)
+    return "edited integration {integration} for project {project}".format(**audit_log_entry.data)
+
+
+def render_integration_remove(audit_log_entry: AuditLogEntry):
+    if audit_log_entry.data.get("provider"):
+        return "uninstalled {name} for the {provider} integration".format(**audit_log_entry.data)
+    return "disabled integration {integration} from project {project}".format(
+        **audit_log_entry.data
+    )
+
+
+def render_internal_integration_add(audit_log_entry: AuditLogEntry):
+    integration_name = audit_log_entry.data.get("name") or ""
+    return f"created internal integration {integration_name}"
+
+
+# Register the AuditLogEvent objects to the `default_manager`
+default_manager.add(
+    AuditLogEvent(
+        event_id=1,
```

Comment:
```suggestion
        event_id=AuditLogEntryEvent. MEMBER_INVITE,
```

@mbauer404 We might be able to avoid usage of magic numbers for `event_id` by making use of `AuditLogEntryEvent`:

https://github.com/getsentry/sentry/blob/9a9b7741624ba507736b5849af817974d5ad0abf/src/sentry/models/auditlogentry.py#L42

Reply:
@dashed The AuditLogEntryEvent class is going to be removed in a future PR and the AuditLogEvent will be replacing it